File: Results/DER/Translation/translation_output/w_test/codeqwen/avatar/python/java/codeforces_334_A.java
Here is the Java version of your Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int i = 1, j = n * n;
        while (i < j) {
            System.out.println(i + " " + j);
            i++;
            j--;
        }
    }
}
```

This Java code does the same thing as your Python code. It reads an integer from the user, then uses a while loop to print pairs of numbers, starting from 1 and ending at `n * n`. The numbers are printed in the format `i j`, with `i` and `j` separated by a space. The loop continues until `i` is no longer less than `j`.

File: Results/DER/Translation/translation_output/w_test/codeqwen/avatar/java/python/codeforces_334_A.py
Here is the Python equivalent of the Java code:

```python
import sys

def main():
    n = int(sys.stdin.readline())
    square = n * n
    odd = [i for i in range(1, square + 1, 2)]
    even = [i for i in range(2, square + 1, 2)]
    
    loop = square // n
    div = loop // 2
    
    for i in range(1, loop + 1):
        for j in range(div):
            if i % 2 == 1:
                sys.stdout.write(str(odd.pop(0)) + " " + str(even.pop(-1)) + " ")
            else:
                sys.stdout.write(str(even.pop(0)) + " " + str(odd.pop(-1)) + " ")
        sys.stdout.write("\n")

if __name__ == "__main__":
    main()
```

This Python code does the same thing as the Java code. It reads an integer from the standard input, creates two lists of odd and even numbers, and then prints out the numbers in a specific pattern. The `pop` method is used to remove and return the last element of the list, which is equivalent to the last element of the array in Java. The `write` method is used to write to the standard output, which is equivalent to `System.out.println` in Java.
